### PR TITLE
Use custom body for the export recordings endpoint

### PR DESCRIPTION
**Pull request by iursevla**
## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [x] Code quality improvements to existing code
- [x] Documentation Update

## Additional information

- This PR fixes or closes issue: fixes https://github.com/blakeblackshear/frigate/pull/13871#issuecomment-2453063035

## Checklist

- [x] The code change is tested and works locally.
- [ ] Local tests pass. **Your PR cannot be merged unless tests pass**
- [x] There is no commented out code in this PR.
- [x] The code has been formatted using Ruff (`ruff format frigate`)

### CODE REVIEW

File: frigate/api/defs/request/export_recordinds_body.py [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+from pydantic.json_schema import SkipJsonSchema
+
+from frigate.record.export import PlaybackFactorEnum
+
+
+class ExportRecordingsBody(BaseModel):
+    playback: PlaybackFactorEnum = Field(
+        default=PlaybackFactorEnum.realtime, title="Playback factor"
+    )
+    source: str = "recordings"
```
Comment (NickM-27) [inline frigate/api/defs/request/export_recordinds_body.py:13]: this is an enum, `PlaybackSourceEnum`

File: frigate/api/defs/request/export_recordinds_body.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    name: str = Field(title="Friendly name", min_length=1, max_length=256)
```
Comment (NickM-27) [inline frigate/api/defs/request/export_recordinds_body.py:14]: Please be more careful when defining these, `name` can be empty and is optional

Comment (iursevla) [reply frigate/api/defs/request/export_recordinds_body.py:14]: Yeah, looking back your're right. Will change

**Review by NickM-27 — APPROVED** [review]

File: frigate/api/defs/request/export_recordinds_body.py
Comment (leccelecce) [inline frigate/api/defs/request/export_recordinds_body.py:1]: Small typo in the filename `frigate/api/defs/request/export_recordinds_body.py`

**Pull request merged**